### PR TITLE
Fix formatting of omitted arguments on their own line

### LINKED ISSUE #5207: Omitted argument on it's own line formatted poorly
```VB
Class C
    Private Sub AAA _
        (ByVal A1 As String _
        , Optional ByVal A2 As String = Nothing _
        , Optional ByVal A3 As String = Nothing
        )

    End Sub
    Sub Main()
        Dim C2 As String = "aaa"
        Call AAA( _
            C2, _
            , _
            C2 _
            )
    End Sub
End Class
```

Pretty listing will force the comma for the omitted argument to column zero.

### PR DESCRIPTION
**Pull request by Pilchie**
Fixes #5207

### CODE REVIEW

**Comment by Pilchie** [comment]
Tagging @dotnet/roslyn-ide and @heejaechang for this version

File: src/Workspaces/VisualBasic/Portable/Formatting/Rules/AdjustSpaceFormattingRule.vb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 Imports System.Composition
+Imports System.Threading
```
Comment (Pilchie) [inline src/Workspaces/VisualBasic/Portable/Formatting/Rules/AdjustSpaceFormattingRule.vb]: Removed

**Comment by heejaechang** [comment]
:+1:

**Comment by Pilchie** [comment]
retest this please

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by Pilchie** [comment]
@dotnet-bot test this please

**Comment by Pilchie** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@Pilchie Hang on one sec, I'm still resolving the issue.

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by tannergooding** [comment]
test this please

**Pull request merged**